app_dart/lib/src/model/appengine/github_build_status_update.dart

```diff
@@ -39,6 +40,9 @@ class GithubBuildStatusUpdate extends Model {
   @IntProperty(propertyName: 'Updates', required: true)
   int updates;
 
+  @IntProperty(propertyName: 'UpdateTimestamp')
+  int updateTimestamp;
```

Comment:
It's better to explain the purpose: "The last time when the status is applied to the PR".

Nit: rename it `appliedTimeMillis` to indicate the unit is millisecond.

Reply:
Added doc., and change name to `updateTimeMillis` (seems update is more appropriate?).